scripts/helm/install

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/../.." || return; pwd)"
+
+(
+  cd "${projectRoot}"
+
+  echo "Deploying claim message queue"
+  helm repo add activemq-artemis https://mailrucloudsolutions.github.io/activemq-artemis-helm/
+  helm upgrade \
+    --atomic \
```

Comment:
I get the error `Error: unknown flag: --atomic` when running this script.
Helm versions are client v2.9.1, server v2.14.3

Reply:
`--atomic` was added in [v2.13.0](https://github.com/helm/helm/releases?after=v2.13.1).

I just had to reinstall Helm after a Docker Desktop update and it put me on v2.14.3 for both client and server.

Reply:
strange they are so out of sync, a forced upgrade fixed it, thanks